docker-compose.build.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
```

Comment:
This is probably a larger discussion and not specific to this change set but are we still actively supporting Windows 2019 & Ubuntu 18.04 images? Windows is not compatible with with different versions so it makes some sense to support both 2019 & 2022, but Ubuntu is cross compatible so I'm not sure if there's a need to support 18.04 any longer. The release pipeline for 18.04 has been removed so I think we may be able to do away with 18.04 here.

Reply:
I'm happy with either. I only put all of them in there just in case. The 18.04 folder is still there though. We'll want to clean it up properly to avoid confusion.

If we ever need to release a windows only version, say a patch for win2019, it gets a little hairy. The new pipeline works under the assumption that windows and ubuntu will always have the same versions to support the combined manifest. If windows-only releases are a requirement we need to tweak things a little.